src/actions/tests/__snapshots__/preview.spec.js.snap

```diff
@@ -46,14 +46,7 @@ exports[`setPreview react instance 1`] = `
 Object {
   "cursorPos": undefined,
   "expression": "this",
-  "extra": Object {
```

Comment:
After I updated `inComponent.js` this got empty. I don't think this should be the right snapshot. I'll investigate later.

Reply:
Yep -- that's an issue.

Reply:
Thanks @darkwing. I will review it this week. :)